columnphysics/icepack_therm_vertical.F90

```diff
@@ -1154,7 +1157,8 @@ subroutine thickness_changes (dt,        yday,     &
          wk1         , & ! temporary variable
          zqsnew      , & ! enthalpy of new snow (J m-3)
          hstot       , & ! snow thickness including new snow (m)
-         Tmlts           ! melting temperature (deg C)
+         Tmlts       , & ! melting temperature (deg C)
+         sblx            ! flux error due to cond/sub inconsistency
```

Comment:
what do the letters in the symbol `sblx` represent?  I prefer to use intuitive names for things, even if they are highly abbreviated.

Reply:
I think ```sblx``` was initially coined to be sublimation flux, i.e., the energy difference introduced by sublimation/condensation. Sorry, it is not a particularly good name. Any suggestions are welcome.

Reply:
100% agree. Will fix.